api/pb/javascript/package.json

```diff
@@ -15,11 +15,11 @@
     "threads_pb_service.d.ts"
```

Comment:
Ah right, so this is going to kick off a new javascript client package. We'll have to make sure we update downstream JS clients as well.

Reply:
yep, i have one more change coming (different PR) and then will do some testing with `textile`. if all goes well, we can do a release and that will kick off these low-level js libs.